fetch-rum-insights/common/aggregations.js

```diff
@@ -0,0 +1,150 @@
+function calculateSessionDurationP50(chunks) {
```

Comment:
we don't use most of these aggregations, but just kept as-is from the rum-insights project for now